climada/util/plot.py

```diff
@@ -210,36 +179,56 @@ def geo_scatter_from_array(array_sub, geo_coord, var_name, title,
 
     if 'cmap' not in kwargs:
         kwargs['cmap'] = 'Wistia'
+
     if axes is None:
-        _, axes = make_map(num_im, proj=proj, figsize=figsize)
-    axes_iter = axes
+        proj_plot = proj
+        if isinstance(proj, ccrs.PlateCarree):
+            # use different projections for plot and data to shift the central lon in the plot
+            xmin, xmax = u_coord.lon_bounds(np.concatenate([c[:, 1] for c in list_coord]))
+            proj_plot = ccrs.PlateCarree(central_longitude=0.5 * (xmin + xmax))
```

Comment:
Would this code work if instead of `ccrs.PlateCarree` one uses another projection from `ccrs` that uses lat/lon?

Reply:
There is no other projection that has lat/lon as output type because that is the unique defining property of the "Equidistant Cylindrical (Plate Carrée)" projection: https://proj.org/operations/projections/eqc.html 

Note that, according to the docstring, the `proj` attribute specifies the CRS of the data. There is currently no way to specify the projection of the axes. To achieve that, the user has to provide an own axes object using the `axes` parameter.

What you might have intended with your question: If the user has data for Fiji in some CRS other than lat/lon (like UTM), and the user doesn't specify any `axes`, we will probably end up with a plot that doesn't have the "nice" central longitude. However, this is at the same time very hypothetical AND hard to deal with. I strongly suggest to leave this for a discussion in the CRS task force.